iBudget/fund/views.py

```diff
@@ -50,13 +55,18 @@ def show_fund_by_group(request):
 
     user = request.user
     users_group = []
+    icon_if_none = \
+        'https://s3.amazonaws.com/family-finance-tracker-static/standard_fund/funds.png'
```

Comment:
And this one is a duplication of hardcoded string that is even worse :)

Reply:
no problem, I will correct!